tests/bluetooth/tester/boards/nrf5340dk_nrf5340_cpuapp.conf

```diff
@@ -0,0 +1,23 @@
+# CONFIG_TEST enforces minimal logging, which we don't want
```

Comment:
@carlescufi Why don't we want minimal logging? 
Or why does TEST_DEFAULTS select minimal logging for that matter?

Reply:
`TEST_DEFAULTS` selects minimal logging to try and fit as many tests as possible in the smallest devices. We don't want minimal logging here because we have enough memory to handle full logging.